### PR TITLE
Create homepage with "pluggable" bands for content

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

**Review by cybette — DISMISSED** [review]

**Review by GregSutcliffe — APPROVED** [review]
Builds and looks good. Seems like a solid base to start from!

File: README.md [hunk 1]
```diff
@@ -20,10 +20,20 @@ questions.
...
-
 Configuration file for the site is ``conf.py``.
 
+### Requirements
+
+- Visit [Nikola installation documentation](https://getnikola.com/getting-started.html#install).
+- Install `sass` to compile scss files to css.
+
+You can install `sass` as described in the [Sass install docs](https://sass-lang.com/install) or:
+
```
Comment (maxamillion) [inline README.md:31]: I would mention here that these are instructions specific for [Fedora](https://getfedora.org/).

**Review by maxamillion — COMMENTED** [review]
One small comment, otherwise LGTM!

File: README.md (hunk 1 above)
Comment (oraNod) [reply README.md:32]: Thanks I've updated to point that out.

**Review by cybette — DISMISSED** [review]

**Review by GregSutcliffe — APPROVED** [review]

**Review by GregSutcliffe — APPROVED** [review]

**Review by cybette — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Merged, huge thanks @oraNod!

**Pull request merged**